README.md

```diff
@@ -1,5 +1,18 @@
 ### Arrays
 
+#### First
+
+* Underscore
+  ```javascript
+  _.first([5, 4, 3, 2, 1]);
+  ```
+
+* ES2015
+
+  ```javascript
+  const [n] = [5, 4, 3, 2, 1];
+  ```
+
```

Comment:
Maybe it's not the best example. The point I tried to get across in the blog post is that you can often just write
```javascript
array.map(x => x[0])
```
instead of
```javascript
_.map(_.first)
```
If we define our own `first` function, people will come to tell we're now implementing an untested, ad hoc implementation of half of Underscore. :wink: